content/events/agbt-2023.md

```diff
@@ -0,0 +1,30 @@
+---
+author: "AnVIL"
+conference: "AGBT"
+description: "Advances in Genome Biology and Technology (AGBT) delivers a premier experience where heads of labs, institutions, businesses, financial analysts and other high-level stakeholders come together to advance the field and drive game-changing innovation."
+eventType: "Conference"
+featured: true
+location: "Hollywood, FL, USA"
+sessions: [{sessionStart: "6 Feb 2023"}, {sessionStart: "7 Feb 2023"}, {sessionStart: "8 Feb 2023"}]
+timezone: "America/New_York"
+title: "Advances in Genome Biology and Technology 2023"
+---
+
+<event-hero></event-hero>
+
+### Description
+
+Project Director Michael Schatz will present a talk describing the AnVIL Project and how it has enabled advances in computational genomics and genome biology research.
+
+### Background
+Advances in Genome Biology and Technology (AGBT) delivers a premier experience where heads of labs, institutions, businesses, financial analysts and other high-level stakeholders come together to advance the field and drive game-changing innovation. Significant scientific advances are announced and showcased, science and industry forge enduring partnerships, and significant investments and notable acquisitions are made.
+
+### Prerequisites (if any)
+
+Register at the [conference webpage] (https://www.agbt.org/events/general-meeting/register-submit-abstract/).
```

Comment:
Hi @jpaten the link here renders as per below:

<img width="791" alt="image" src="https://user-images.githubusercontent.com/18710366/213630825-4b792572-a1e9-404d-b345-d878557866d3.png">

Would you mind taking a look (I suspect there is a space between the `[]` and the `()`.